test/ui/views/more_view_test.dart

```diff
@@ -0,0 +1,45 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+// VIEW
+import 'package:notredame/ui/views/more_view.dart';
+
+import '../../helpers.dart';
+
+void main() {
+  group('MoreView - ', () {
+    setUp(() async {
+      setupNavigationServiceMock();
+      setupCacheManagerMock();
+    });
+
+    tearDown(() {});
+
+    group('UI - ', () {
```

Comment:
Add some test to check the navigation when you tap on each list tile :slightly_smiling_face:

Reply:
done